rover_msgs/ZedGimbalData.lcm

```diff
@@ -0,0 +1,6 @@
+package rover_msgs;
+
+struct ZedGimbalCmd 
```

Comment:
This doesn't seem to match the filename - should this be ZedGimbalCmd?

Reply:
Nope but the struct should be ZedGimbalData. James you are making a pull request for the ZedGimbalCmd, right?